tests/conftest.py

```diff
@@ -1,4 +1,6 @@
 from re import findall
+from sys import path
+path.append("/app")
```

Comment:
It's possible that you're seeing that you need this because you're missing a `__init__.py` file in the `tests` folder - if you try adding an empty file named precisely that (two underscores each side) then that should ensure the `tests` folder is recognised as a module, and should allow imports - you will need to tweak the conftest imports to become `from test.conftest ...` as well